### DISCUSSION TITLE
Panthor and mainstream mesa/linux

### DISCUSSION THREAD

**Discussion by stevepringle** [Joshua-Riek/ubuntu-rockchip#518] [General]
It looks like it wont be too long now before panthor is merged in to  mainstream mesa.
(https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/26358). Once the merge is completed, would it be possible to produce a release based on that and mainstream Linux 6.7 rc5 or later with desktop support?

**Comment by Joshua-Riek** [comment]
It would be possible, but it still has a lot of bugs, I would prefer to wait until all of this is merged. Recently, I have been doing a lot of work on the 5.10 kernel, and I am soon porting everything over to the new 6.1 BSP (not yet released). For now, I prefer to focus on a kernel that has full hardware support, but I'm sure I will get to this in the future.

**Reply by stevepringle** [reply]
Understood,I think you are doing an amazing job. Keep up the good work :-)
